index.html

```diff
@@ -0,0 +1,196 @@
+                <h3>CHOOSE YOUR PRICING PLAN</h3>
+                <p class="text">Pick any of our super affordable pricing plans</p>
+            </header>
+            <section>
+                <div class="container5-section2">
+                    <div class="container5-section2--items">
+                        <h5>BASIC</h5>
+                        <p class="price">$15</p>
+                        <p class="text-small">per month</p>
+                        <p class="text-align text-bold">
+                            Up to 7 Projects<br>
+                            2 Additional Developers
+                        </p>
+                        <button class="button-container5">Get Started</button> 
+                    </div>
+                    <div class="container5-section2--items">
+                        <h5>AGENCY</h5>
+                        <p class="price">$55</p>
+                        <p class="text-small">per month</p>
+                        <p class="text-align text-bold">
+                            Up to 25 Projects<br>
+                            2 Additional Developers<br>
+                            Unlimited Support
+                        </p>
+                        <button class="button-container5">Get Started</button>
+                    </div>
+                    <div class="container5-section2--items">
+                        <h5>PRO</h5>
+                        <p class="price">$75</p>
+                        <p class="text-small">per month</p>
+                        <p class="text-align text-bold">
+                            Up to 25 Projects<br>
+                            2 Additional Developers<br>
+                            Unlimited Support<br>
+                            1.5GB Disk Space
+                        </p>
+                        <button class="button-container5">Get Started</button>
+                    </div>
+                </div>
+            </section>
+        </div>
+    </div>
+    <div class="container section-6">
+        <div class="content-wrapper">
+            <header class="container6-header">
+                <h1 class="h1-special">THE TEAM</h1>
+            </header>
+            <section>
+                <div class="container6-section2">
+                    <div class="container6-section2--items">
+                        <img src="./images/cto.png" alt="image CTO" class="profile-img">
+                        <p class="text-large">CTO</p>
+                        <p class="text-bold">Johnny B Good</p>
+                        <p>The brain behind the whole operation</p>
+                        <div class="container6-icons">
+                            <img src="./images/team-twitter.svg" alt="twitter icon" class="icon-small">
+                            <img src="./images/team-facebook.svg" alt="fb icon" class="icon-small">
+                        </div>
+                    </div>
+                    <div  class="container6-section2--items">
```

Comment:
Jeśli chce Tworzyć nazwy klas zgodnie z BEM (tutaj szczegóły: https://devmentor.pl/b/metodologia-bem-w-css-i-sass) to razem z Modyfikatorem musi wystąpować Element czyli tutaj powinno być: `container6-section2 container6-section2--items`